crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -68,12 +61,39 @@ public Transform Viewpoint
             }
         }
 
-        public Transform Root { get; private set; }
-
+        [Tooltip("The camera which drives the ocean data. Defaults to main camera."), SerializeField]
+        Camera _camera;
+        public Camera ViewCamera
+        {
+            get
+            {
 #if UNITY_EDITOR
-        static EditorWindow _lastGameOrSceneEditorWindow = null;
+                if (_followSceneCamera)
+                {
+                    var sceneViewCamera = EditorHelpers.GetActiveSceneViewCamera();
+                    if (sceneViewCamera != null)
+                    {
+                        return sceneViewCamera;
+                    }
+                }
 #endif
 
+                if (_camera != null)
+                {
+                    return _camera;
+                }
+
+                return Camera.main;
```

Comment:
Should this set `_camera = Camera.main` to cache it before returning?

Reply:
I have removed setting the properties. We will now work directly from _Camera.main_. If anyone wants the performance benefit, they will have to set the properties themselves. I've added a warning for below 2019.4.9 since performance for _Camera.main_ has been greatly improved in that version. I have added some local caching here and there for when the camera property not set where it makes sense.

The reason being is that now camera switching will work out-of-the-box providing the _MainCamera_ tag is used.